### DISCUSSION TITLE
Community meeting agenda for NA/EU 2023-04-25

### DISCUSSION THREAD

**Discussion by Naomi-Wash** [openmetaverse/community#26] [Meetings]
**Meeting Details**

- Date/Time: 24 April 2023 @ 1:00pm ET / 6:00pm UTC
- Location: [Community Voice Channel](https://discordapp.com/channels/948320633522114570/1067216331474210856)
- Moderator: Bernelle Verster
- Note Taker: TBD

The [Community Meetings](https://github.com/Open-MV/community/tree/main/meetings) repo contains the history of past meetings, including a link to the agenda, recording, notes, and resources.

We will hold our weekly meetings on the [OMF Discord Community Channel](https://discord.gg/openmetaverse)
Comments and ideas can be posted to the voice text area or in the Community channel

If you’re an observer or here for the first time, this is a meeting where the OMF community comes together to discuss updates and contribution opportunities for the foundation and its projects. This is your opportunity to become a voice of influence in the community!

Please introduce yourself in #Community or the voice channel chat as we dive into these agenda items. There will be an open floor in the last 15 minutes of the meeting (if possible) where anyone can discuss items that they feel are relevant to our current or near-future goals.

### Agenda

**Community Updates**

**Update on meetings**

**Open Floor**
Outcomes from Discussion topics; What have you seen in the past week that is interesting?

**Action Items**
Create actionable items from proposed topics - after the meeting

**Open Discussion Items for next meeting**

List any additional items below!

**Comment by indiebio** [comment]
Reflecting on engagement: What do you need to make OMF more relevant to you? Should the meetings look different? What do you want from OMF? How can we integrate what happens here to be more relevant to what you do, what you need and what you want to contribute?

**Comment by indiebio** [comment]
Master hedgedoc with the links to the other FIG hedgedocs: https://hedgedoc.openmv.org/DY_RMofxSOeMF_Lo1su__w

Progress on these FIGs! 
Unpacking the [narrative](https://docs.google.com/document/d/1BwugBRX5DaYA-SF32eT-abDfwH0tJWE28hdpYKvZ5iw/edit) point by point as it relates to:
Networking | https://hedgedoc.openmv.org/VvbBD6SpRcavcmlEbLub6w
Security | https://docs.google.com/document/d/1NO5DCBRD8IZa9UXfluvaExwdirqt3p1z05Vpaet15wk/edit 
  - lesssneaky, can we move this into a hedgedoc? and micky then can add it to the master doc. 
  - Ideally it would be best if these links can be added to the [github overview table](https://github.com/openmetaverse)

**Comment by indiebio** [comment]
In the last digital assets meeting and chat there was talk about how to disseminate information better. This is both visualising or summarising written content, and summarising the voice meetings. AI was suggested to convert the [voice meeting](https://github.com/openmetaverse/fig-digitalassets/blob/main/meetings/readme.md#Previous-Meetings) to text ([see the generated summary of that meeting](https://raw.githubusercontent.com/openmetaverse/fig-digitalassets/main/meetings/minutes/20230410-digitalassetssummary.md)), but in my opinion this was unsuccessful.
In this meeting some alternatives were discussed.
I think the quality of the content (higher signal-to-noise (general or repetitive chats) ratio) and setting a more structured agenda may help more than technological tools.

**Reply by indiebio** [reply]
So my totes biased notes from the digital assets meeting:
 - [AssemblyAI](https://www.assemblyai.com/) - transcribe podcast with speaker names and tags attached. Can then send that to ChatGPT which can summarise it,in theory meaningfully.
 - MSF uses [FireFlies.ai](https://fireflies.ai/) which includes synchronised hyperlinks with a summary and an outline, like a thorough dashboard. And something about including zoom comments.
 "Useful breadcrumbs between meetings" needed - perhaps this github page for the community meeting is a good place. Manual transfer with a real human seems to be needed, but anyone can do it! 
 
 08:30 min: Having a broad overview of what the group has been discussing is helpful - summary of previous meeting, past discussions, a visual image, a poster, infographic, a presentation...
  - [Infinite canvas tools](https://infinitecanvas.tools/gallery/) unlock the visual thinking
  - TLDraw
  - Figma (check the spelling on these)
  - (in draft, ran out of time, feel free to add comments)
 
 Other thoughts:
 - Transcription of the meetings are useful to help decipher accents, for example. 
 - The chat section of the meetings are also great and high signal-to-noise.

**Reply by indiebio** [reply]
18:42: collaboration, like a game engine
 - [PlayCanvas](https://playcanvas.com/) - like multiplayer Unity

**Reply by indiebio** [reply]
Markdown stuff

- [Obsidian](https://obsidian.md/) for organising markdown.
- [Hackmd](https://hackmd.io/) for collaborating in markdown.
- [Hedgedoc on the openmv server](https://hedgedoc.openmv.org/) which is poor at indexing, finding things again.

https://huggingface.co/datasets/allenai/objaverse
https://objaverse.allenai.org/explore/

**Reply by indiebio** [reply]
51 minutes, something about GLV and thumbnail indexing. I think the conversation is about using e.g. https://objaverse.allenai.org/explore/ and then something about listing the content as thumbnails. and the content here is not just a digital asset, but it's a specific information rich thing that shows the progress of the group and information on the projects

**Comment by indiebio** [comment]
From jin: i wrote some scripts relevant to converting github discussions to posters that can be pinned to 3D spaces
https://github.com/madjin/github-memex

**Comment by indiebio** [comment]
In the community meeting, jin showed indiebio (me) how to enter and navigate the virtual space jin created for [MSF working groups](https://hyperfy.io/MSF/). There is an as yet empty space for the OMF digital assets FIG. 

jin illustrated pasting images and weblinks - even interactive ones - into the virtual space, to create an exhibition space.

My reflections on using a virtual space for having an immersive, visual overview of progress in any particular FIG or the OMF project as a whole:

- Learning how to be inside this is not trivial. New users may need a demo. It is intimidating to people not used to playing games like this - my "go-to" stereotype is non-IT technical, middle-aged women (aka, me). This does not mean we can't do it, but facilitating participation is critical. A demo, sign-post images in-world, a guided walk-through with an actual person, like jin kindly did for me, and avatar choice needs to be considered.

Overte has weekly events, we could have a weekly casual event where we guide new people through and chill and build together. See e.g. https://calendar.overte.org/

- Navigating is hard; WASD is not natural for non-shooter players. I just get the hang of it, and moving my mouse, then I need to multi-task doing something else and remember to press escape to get to my text file or email or whatever and it all gets frustrating and overwhelming fast. Again, this is not a deal-breaker, I imagine it won't take too much time to get used to this, but for this reason I strongly recommend not having meetings in here. I am sure we can have joint meetings between the virtual space and Discord sharing an audio and screenshots where needed. The meetings do need to be recorded, is this possible?

 - The computing power needed may be a problem if someone wants to join on the road or on mobile, but if this can be bridged via audio only and recorded this is doable, in my opinion.

 - The avatar freaks me out, it feels like I suddenly have to be aware of what my body does and who its close to and who is close to me and it's just too much to think about. I imagine it is possible to simply be an orb or a cube or something not relatable or harassable at all. 

 - We need to be careful to not just bring the bad of the real world into the virtual. Why does this have to look like an office block? Does it have to feel like a conference space? We do need some things to relate to, but we can make it better - I want to sit in a garden for example (in real life the noise and the solar glare prevents this)

**The things I do like:**

- It's really nice to be able to scan over everything and get a visual of what is effectively a 3D dashboard.

- This does not solve the primary problem of engagement, we still need people to visualise progress in an appropriate and engaging way, to design well and beautifully, have it not look like a boring academic conference, but the potential for beautifully curated knowledge spaces may already inspire people to contribute - I am inspired!

 - The visuals are linkable so one can still hyperlink to the original content. We can have interactive sites so collaboration is possible, I think this works better than in Discord for many things. I liked the post-it board jin shared, but I want to be able to move the post-its and add new ones - jin's example was a screenshot hyperlinked to the original but I'm sure we can find embeddable ones.

- Having interactions in the virtual space can really work well but I would still not want to have proper meetings in here. So instead of the more formal meetings, we agreed that it may be more welcoming to rename interactions in the virtual world to "lounge". We can also have several spaces for different types of conversations - water cooler, hallway, garden, ferris wheel, whatever.

**Comment by indiebio** [comment]
Being convinced that a virtual space is a good idea, the next question is the choice of virtual world:

**[Hyperfy](https://hyperfy.io/MSF/)**
jin chose to work in hyperfy. His reasons are that it works in the browser, one doesn't have to download anything. It is easy to drag things in and create/spawn things. To create things one does need to connect with a wallet, but anyone can view. I don't like this aspect, but jin has already put in a lot of work here.

Another world jin showed was [FrameVR](https://framevr.io/omi). It has a different "feel" to it. It does not seem to require a wallet.

I don't know how FOSS these are. My favourite choice would be [Overte](https://overte.org/downloads.html), as it is proper open source, no wallet required. It  needs desktop installation (Linux or Windows) however, so harder to just dip in and look around. Vircadia is a fork which will have web functionality but they're going the blockchain route.

The worlds have portals, so it is entirely possible to use whatever virtual world you like and then put a portal in to connect worlds. 

**ACTION:** We need to decide as a group where our entry virtual space is, the one we link to from our website and social media, and where we can send complete beginners to. That needs to be one that is easy to dip into and view, without an account. From a PR perspective it would have to be compatible with Linux Foundation and OMF values.

Then jin shared some visual tools, or infinite canvases that allow collaborative visual work. 
- [Figma](https://www.figma.com/) - 
- I use [Milanote](https://app.milanote.com/)
- Also see [Infinite canvas tools](https://infinitecanvas.tools/gallery/) and TLDraw from comment above.

We don't have to commit to any one of these as they are ported into the virtual space via an iframe, so whatever (combination of tools) work for people can be used.
In my experience (maybe just my messy head) these visual interfaces confuse the team over time, so they're only really good for shorter meetings or brainstorms, and still need to be processed and archived afterwards, but that's a separate issue.

**Reply by indiebio** [reply]
Others - Tara D works on this: https://www.3rdrockgrid.com/blank-page
